src/Shared/BrowserTesting/src/BrowserKind.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.BrowserTesting
+{
+    [Flags]
+    public enum BrowserKind
+    {
+        Chromium = 1,
+        Firefox = 2,
```

Comment:
I know it doesn't matter but why is `3` skipped?

Reply:
It's a flags enum, so you want to be able to target individual bits